packages/filter-workspace-packages/package.json

```diff
@@ -35,6 +35,7 @@
     "execa": "^4.1.0",
     "find-up": "^5.0.0",
     "is-subdir": "^1.1.1",
+    "minimatch": "^3.0.4",
```

Comment:
use micromatch instead, as we already have it in deps

Reply:
done [here](https://github.com/pnpm/pnpm/pull/3035/commits/7ab5f6d4dbd661c6113cefa3bfe3e955478793b0)